cmake/CodeCoverage.cmake

```diff
@@ -0,0 +1,748 @@
+# Copyright (c) 2012 - 2017, Lars Bilke
+# All rights reserved.
+#
```

Comment:
We need LICENSE+NOTICE changes for this. Please check which license is below, and add its name to LICENSE as well.